test/unit/operation_test.js

```diff
@@ -19,9 +19,9 @@ describe('Operation', function() {
 
     describe(".payment()", function () {
         it("creates a paymentOp", function () {
-            var destination = "gspbxqXqEUZkiCCEFFCN9Vu4FLucdjLLdLcsV6E82Qc1T7ehsTC";
+            var destination = "GB7ANT5Q6PZ6KJGAXGQ63ZX3TM7BBLANJTUJ4SQOAHRVWQL6HNVJPYLZ";
```

Comment:
No, I just assumed that since tests are independent it doesn't matter. Am I wrong?